app/questionnaire_state/relationship_state_question.py

```diff
@@ -15,8 +16,8 @@ def build_repeating_state(self, user_input):
         template_answer = self.answers.pop()
         group_instance = template_answer.group_instance
 
-        first_name_answers = get_answer_store(current_user).filter(answer_id='first-name')
-        last_name_answers = get_answer_store(current_user).filter(answer_id='last-name')
+        first_name_answers = get_answer_store(current_user).filter(answer_id='first-name')[0:settings.EQ_MAX_NUM_REPEATS]
```

Comment:
This sort of chaining of methods followed by an array index seems pretty dirty to me. Can we break it into multiple lines, return the answerstore, then names, then choose the subset? 

What might be better is to add a limit flag to answerstore filtering, to end early, rather than looping through and fetching everything if we're going to throw anything above an index of 25 away.